action.yaml

```diff
@@ -48,32 +48,40 @@ inputs:
 runs:
   using: composite
   steps:
+    - name: Install uv
+      uses: astral-sh/setup-uv@v3
+
+    - name: Set up environment
+      run: uv venv
```

Comment:
I wonder if we should pass `--python=X.Y` here to avoid a floating version of Python.

Reply:
From what I see we don't do that currently so happy to skip for now, but mentioning it since it stood out to me - I'm newer to Python though so this may or may not make sense here.